.gitignore

```diff
@@ -0,0 +1 @@
+ng
```

Comment:
I use `go install neugram.io/ng` for building the binary, so I'm never left with one floating around the tree. I'd rather not encourage a build process that does.